frontend/src/component/personalDashboard/Grid.tsx

```diff
@@ -1,6 +1,74 @@
 import { Box, Grid, styled } from '@mui/material';
 import type { Theme } from '@mui/material/styles/createTheme';
 
+export const ContentGridContainer = styled('div')({
+    containerType: 'inline-size',
+});
+
+const ContentGrid2 = styled('article')(({ theme }) => {
+    return {
+        backgroundColor: theme.palette.divider,
+        borderRadius: `${theme.shape.borderRadiusLarge}px`,
+        overflow: 'hidden',
+        border: `0.5px solid ${theme.palette.divider}`,
+        gap: `2px`,
+        display: 'grid',
+
+        '&>*': {
+            backgroundColor: theme.palette.background.paper,
+        },
+    };
+});
+
+export const ProjectGrid = styled(ContentGrid2)(({ theme }) => ({
+    gridTemplateAreas: `
+            "title"
+            "onboarding"
+            "projects"
+            "box1"
+            "box2"
+            "owners"
+            `,
+
+    '@container (min-width: 1000px)': {
+        gridTemplateColumns: '1fr 1fr 1fr',
+        gridTemplateAreas: `
+                "title onboarding onboarding"
+                "projects box1 box2"
+                ". owners owners"
+            `,
+    },
```

Comment:
because we use named areas in the items, that messes things up if we suddenly don't have any areas mentioned in the 1-column layout. As such, the workaround is to list them in all possible layouts.

Reply:
Optionally (just thought of this): we could only use a grid layout when we're wider than X px, and use a flex layout otherwise. That's probably a cleaner option.

Reply:
I think I prefer the grid/flex distinction depending on size